go/tasks/pluginmachinery/core/resource_manager.go

```diff
@@ -4,21 +4,23 @@ import (
 	"context"
 )
 
-type AllocationStatus string
+//go:generate enumer -type=AllocationStatus -trimprefix=AllocationStatus
+
+type AllocationStatus int
 
 const (
 	// This is the enum returned when there's an error
-	AllocationUndefined AllocationStatus = "ResourceGranted"
```

Comment:
isnt this a breaking change?

Reply:
not sure, but do we store this in redis?

Reply:
We do not store this value anywhere as far as I can tell...